command_line/merging_statistics.py

```diff
@@ -131,16 +131,22 @@ def table1_tex(merging_stats):
 def run(args):
     if len(args) == 0 or "-h" in args or "--help" in args:
         master_params.show()
+        sys.exit()
     interp = master_params.command_line_argument_interpreter(args)
     phil_scope, unhandled = interp.process_and_fetch(
         args, custom_processor="collect_remaining"
     )
     params = phil_scope.extract()
-    merging_stats = []
+    input_files = []
 
     for arg in unhandled:
-        assert os.path.isfile(arg)
-        merging_stats.append(merging_statistics.run([arg], master_params=phil_scope))
+        if not os.path.isfile(arg):
+            print(f"{arg} is not a file or interpretable as a parameter")
+            sys.exit(1)
```

Comment:
```suggestion
            sys.exit(f"{arg} is neither a file nor an interpretable parameter")
```
also makes the message go to stderr